web/frontend/src/pages/form/components/AddQuestionModal.tsx

```diff
@@ -132,6 +133,9 @@ const AddQuestionModal: FC<AddQuestionModalProps> = ({
         return;
     }
   };
+  const choices = ChoicesMap.ChoicesMap.has(language)
```

Comment:
This might come out as `undefined`. But I guess we suppose that `en` is always present. Is there a test for that in the editing?